grapher/color/ColorScale.ts

```diff
@@ -342,9 +342,15 @@ export class ColorScale {
         })
     }
 
-    getColor(value: number | string | undefined): string | undefined {
+    getBinIndex(value: number | string | undefined): number {
         return value === undefined
-            ? this.noDataColor
-            : this.legendBins.find((bin) => bin.contains(value))?.color
+            ? -1
+            : this.legendBins.findIndex((bin) => bin.contains(value))
+    }
```

Comment:
Is it possible to turn this into `getBin()` instead, which I think returns the formatted label?

Reply:
good pointer! just changed it.